### PR TITLE
Snitch profile

### PR DESCRIPTION
**Pull request by rcaina**
Snitches viewed on user profile, as well as user being able to view snitches of partners and trainees (if user is a trainer)

### CODE REVIEW

File: react-native-app/views/SnitchesView.tsx [hunk 1]
```diff
@@ -10,9 +10,8 @@ import SnitchEvent from '../shared/models/SnitchEvent';
 import User from '../shared/models/User';
 import PaginatedList from '../components/PaginatedList';
 import { UserSnitchesResponse } from '../shared/models/requests/UserSnitchesRequest';
-import SocialShareBtn from '../components/SocialShareBtn';
 
-const PAGE_SIZE = 10
+const PAGE_SIZE = 1
```
Comment (amj311) [inline react-native-app/views/SnitchesView.tsx]: It looks like this number might have been changed for testing purposes? I'm not sure we need the snitch feed page size to be just 1. If it was just a test, let's change it back.

**Review by amj311 — CHANGES_REQUESTED** [review]
This looks great! I only have two small comments. One is in the SnitchesView where it looks like the page size was changed to 1.

The other is that we seem to have two PartnerAssociationRequestButton components, but with different capitalization. Do you think that could be resolved in this PR?

File: react-native-app/views/SnitchesView.tsx (hunk 1 above)
Comment (rcaina) [reply react-native-app/views/SnitchesView.tsx]: Not quite sure why there are 2 PartnerAssociationRequestButton components. When I do a pull on main only one shows up and it says it is up to date. and so same when I merge to this branch. There is only one of those components. Tried to re-commit and all but it says there  is nothing to commit so not sure how to go about getting rid of the extra component. The one that should be kept is the one that is Capitalized at the beginning (PartnerAssociationRequestButton). The other should be deleted

Comment (Filaipus) [reply react-native-app/views/SnitchesView.tsx]: I think that might be my fault (and github?) 
When I pulled main there was that button but with lowercase P, which made Metro go crazy (no idea how the person that did that change was compiling the project). So I changed it to uppercase P. I guess when I merged github didn't recognize it as a name change and instead as two different files and somehow kept both? Not sure, specially not sure about why NOW it doesn't recognize that there are two files haha

**Review by amj311 — APPROVED** [review]

File: react-native-app/views/SnitchesView.tsx (hunk 1 above)
Comment (amj311) [reply react-native-app/views/SnitchesView.tsx]: It's been happening elsewhere, so it's probably not your fault

Comment (amj311) [reply react-native-app/views/SnitchesView.tsx]: > Not quite sure why there are 2 PartnerAssociationRequestButton components. When I do a pull on main only one shows up and it says it is up to date. and so same when I merge to this branch. There is only one of those components. Tried to re-commit and all but it says there is nothing to commit so not sure how to go about getting rid of the extra component. The one that should be kept is the one that is Capitalized at the beginning (PartnerAssociationRequestButton). The other should be deleted

Okay, we can leave it for now and address it later

**Pull request merged**